LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 The Notation Authors
```

Comment:
How do we determine the year? Should it be `2022` (the 1st year of this repo) or `2020` (the 1st year of the notary v2 work)? or even `2019` (the year we kick off the project)?

Reply:
For determining year I looked at the date of [first commit](https://github.com/notaryproject/notation/tree/560d4f8f56546c51ee6c3e5a7241f1961cd8bd78) in Notation and used that for all 3 Notation repos.
PRs for all repo:
https://github.com/notaryproject/notation/pull/536
https://github.com/notaryproject/notation-go/pull/267
https://github.com/notaryproject/notation-core-go/pull/116

Reply:
Shouldn't this actually be?
> Copyright 2020-2022 The Notation Authors

This spans all the years development was in progress.